packages/ui/src/council/modals/VoteForCouncil/VoteForCouncilSuccessModal.tsx

```diff
@@ -21,7 +21,7 @@ export const VoteForCouncilSuccessModal = ({ onClose, candidateId }: Props) => {
   const { candidate } = useCandidate(candidateId)
 
   const goToElection = useCallback(() => {
-    history.push(generatePath(ElectionRoutes.currentElection))
+    history.go(0)
```

Comment:
Is this necessary here to go to the first page of elections?

Reply:
the original code tried to update Election page, but it is restricted by pushing same URI. so I tried to refresh full webpage, so it updates the screen, but not good UX

Reply:
Theoretically, this history.go() command is not needed here, so I will check this when there council votes happened.

Reply:
[pioneer-2](https://pioneer-2-git-fork-mkbeefcake-fix-myvotesisntupdated-joystream.vercel.app/) has shorter cycles for testing.